src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator start work");
```

Comment:
The message 'Excavator start work' could be more informative. It should indicate what type of machine is working. Consider specifying what the excavator is doing, for example, 'Excavator is digging' to provide more context.